pkg/tessera/tessera_test.go

```diff
@@ -27,10 +27,14 @@ import (
 	tessera "github.com/transparency-dev/trillian-tessera"
 )
 
-var tileHash = "81bfc09c412c04da53a1b0ddb94dce48d6a24e9ea58987f7d45a04e8007fb3ca"
+var tileHashHex = "81bfc09c412c04da53a1b0ddb94dce48d6a24e9ea58987f7d45a04e8007fb3ca"
 
 func TestAdd(t *testing.T) {
 	ctx := context.Background()
+	tileHash, err := hex.DecodeString(tileHashHex)
```

Comment:
I like colleen's helpers `DecodeOrDie` for these kind of things.

Reply:
Sorry, missed this, fixing in https://github.com/sigstore/rekor-tiles/pull/292